src/components/Carousel/carouselProps.ts

```diff
@@ -25,7 +25,7 @@ export const carouselProps = {
   // count of items to showed per view
   itemsToShow: {
     default: DEFAULT_CONFIG.itemsToShow,
-    type: Number,
+    type: Number || 'auto',
```

Comment:
Incorrect prop type definition 
```suggestion
    type: [Number, String],
```